docs/identity-platform/certificate-credentials.md

```diff
@@ -98,7 +98,7 @@ In the **App registrations** tab for the client application:
 
 After acquiring a certificate, compute these values:
 
-- `$base64Thumbprint` - Base64-encoded value of the certificate hash
+- `$base64Thumbprint` - Base64-encoded value of the certificate SHA-1 hash
 - `$base64Value` - Base64-encoded value of the certificate raw data
 
```

Comment:
This is a good clarification, but it would be even better to call out that this is a different hash algorithm/fingerprint than the `x5t#S256` value above.

```suggestion
- `$base64Thumbprint` - Base64-encoded value of the certificate SHA-1 hash
- `$base64Value` - Base64-encoded value of the certificate raw data

> [!Important]
> Note that the `$base64Thumbprint` value here is calculated with SHA-1, which is different than the `x5t#S56` value in the JWT header, which is SHA-256.

```

Reply:
Yeah, I'll follow up with the folks who own that manifest. The experience in the portal / maniefest should not use SHA1, at least not for new apps. 

But for now, I agree that this is a correct observation. Maybe you can have it as a "Note"? We do expect ppl to use the portal to upload their cert